docs/develop/01_sdk/02_cookbook/06_upgrading_to_v0_29/01_backwards_compatibility.md

```diff
@@ -0,0 +1,57 @@
+---
+id: v29-backwards-compatibility
+title: Backwards Compatibility / Interoperability with Previous Versions
+---
+
+Beside the breaking code changes we have identified two data format changes which might affect interoperability.
+The didsign.io and w3n.id already accept bare requests for attestation, which are effectively the new Credentials,
```

Comment:
Should maybe link to those websites?

Reply:
To me this bit actually reads like its for internal purposes, so I'm wondering whether it should be on the docs at all? As I wrote in the description I copied the text verbatim bc the ticket seemed to suggest it, but I'm not sure that this is what we want.

Reply:
Yeah we can then remove it.